src/Sarif/ExtensionMethods.cs

```diff
@@ -333,6 +333,10 @@ public static string GetMessageText(this Result result, ReportingDescriptor rule
             if (concise)
             {
                 text = GetFirstSentence(text);
+                if (text.Length > maxLength)
+                {
+                    text = text.Substring(0, maxLength) + "\u2026";
```

Comment:
>\u2026 [](start = 59, length = 6)

You should either directly embed the ellipsis or at least comment this so that readers know what the character is.